lib/hanami/cli/generators/app/slice.rb

```diff
@@ -14,23 +14,24 @@ def initialize(fs:, inflector:)
             @inflector = inflector
           end
 
-          def call(app, slice, slice_url_prefix, context: SliceContext.new(inflector, app, slice, slice_url_prefix)) # rubocop:disable Metrics/AbcSize
-            fs.inject_line_before_last(fs.join("config", "routes.rb"), /end/, t("routes.erb", context).chomp)
+          def call(app, slice, slice_url_prefix, context: SliceContext.new(inflector, app, slice, slice_url_prefix))
```

Comment:
Just wondering: what's the thinking behind building the SliceContext such that it's injectable as a call arg? Given its just reusing all the previous args, is there any reason we couldn't simplify this method's params signature and just initialize the `SliceContext` inside the method body?

Reply:
@timriley The gem needs to be cleaned up. If this isn't a blocker for you, I would not take care of this now.